Makefile

```diff
@@ -1,37 +1,94 @@
+.DEFAULT_GOAL := help
+
+
+## Print this message
+.PHONY: help
```

Comment:
Wow, impressive ! but I think we should put it at the end of the file for an easier readability